travis/.travis.yml

```diff
@@ -29,6 +29,7 @@ rvm:
   - jruby-9.1.7.0 # pin JRuby to this until travis/rvm can install later versions
   - jruby-head
   - jruby-1.7
+  - truffleruby-head
```

Comment:
Non head version only please.

Reply:
I'd really like `truffleruby-head`, because `truffleruby` releases are only once every 3 months. That's very impractical to see if TruffleRuby fixes improve compatibility with RSpec master.
Considering it's in allowed failures, what's the issue with using `truffleruby-head` instead of `truffleruby` for RSpec's team?

Reply:
head builds are too flakey, eventually I'd like no allowed failures on our builds

Reply:
Please look at it from our perspective, these are our builds and this doesn't benefit us in anyway.

Reply:
If you're expecting to develop against builds I once again strongly recommend you go and set your own up, have a suite of common ruby tool test suits and run them in your own test repo.

Reply:
Right, I think once TruffleRuby passes all of RSpec's CIs, we should switch to non-allowed-failures and indeed use a release (`truffleruby`) for stability.
But before that happens, it would be much more useful to test against `truffleruby-head`.

Reply:
I suggest we revisit this when you have compatibility.

Reply:
We already tests some gems like RSpec and RSpec generally just works on TruffleRuby, but we can't easily test new changes in RSpec master per commit. Adding it here would provide that.

> I suggest we revisit this when you have compatibility.

We can do that. FWIW I think we only have a couple failures on rspec-support and rspec-core, so it's getting there.

Reply:
I've removed the non-head version and added support information to the readme.